command/validate.go

```diff
@@ -87,7 +88,10 @@ func (c *ValidateCommand) Run(args []string) int {
 	var rawTemplateData map[string]interface{}
 	input := make(map[string]interface{})
 	templateData := make(map[string]interface{})
-	json.Unmarshal(tpl.RawContents, &rawTemplateData)
```

Comment:
It looks like you're totally removing the option to use json files, rather than _additionally_ supporting HCL.  Is that accurate? What would be the cost of continuing to support both for a couple of releases?

Reply:
HCL understands json, this should still work, also all the json tests are still green and there's a bunch.

Reply:
For example when a build file has:
builder.vboxmanage:
```json
 [
        [
          "modifyvm",
          "{{.Name}}",
          "--memory",
          "2048"
        ]
]
```

This is due to the fact